src/ios/CDVWKInAppBrowser.m

```diff
@@ -744,9 +746,7 @@ - (void)createViews
     }
     configuration.applicationNameForUserAgent = userAgent;
     configuration.userContentController = userContentController;
-#if __has_include("CDVWKProcessPoolFactory.h")
-    configuration.processPool = [[CDVWKProcessPoolFactory sharedFactory] sharedProcessPool];
-#endif
+    configuration.processPool = _processPool;
```

Comment:
See [the comment above](https://github.com/sundeepgupta/cordova-plugin-inappbrowser/pull/1/files#r547396659)